### DISCUSSION TITLE
Are Chapter Titles Supported?

### DISCUSSION THREAD

**Discussion by dinkyrdj** [openaudible/openaudible#692] [Q&A] [answered]
Does OpenAudible support chapter titles? Following the suggestion in one of the issues, I checked the book report for a book with known chapter titles (they show in Audible). The book report only shows Chapter 1, etc..., no titles. I checked the documentation page and it makes no mention of chapter titles. Thanks!

**Comment by openaudible** [comment, marked as answer]
Yes! As of 3.1.2!

**Reply by dinkyrdj** [reply]
Great! When do you expect 3.2 to be released?
